vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListFragment.kt

```diff
@@ -152,10 +152,10 @@ class HomeRoomListFragment :
             headersController.submitData(it)
         }
 
-        roomListViewModel.onEach(HomeRoomListViewState::roomsLivePagedList) { roomsListLive ->
-            roomsListLive?.observe(viewLifecycleOwner) { roomsList ->
-                roomsController.submitList(roomsList)
-                if (roomsList.isEmpty()) {
+        roomListViewModel.onEach(HomeRoomListViewState::roomsPagedList) { roomsList ->
+            roomsList?.let {
+                roomsController.submitList(it)
+                if (it.isEmpty()) {
```

Comment:
It's a bit weird to have both call to roomsController, this should be his responsability to handle empty list

Reply:
The problem here is that there are different empty states for different tabs, so we need to force rebuild models, to show different empty state view

Reply:
The controller should be able to handle this regarding the ViewState

Reply:
> there are different empty states for different tabs, so we need to force rebuild models, to show different empty state view

This should be a comment in the code, since this is not usual to call `requestForcedModelBuild()`